### ISSUE TITLE
Nuevo estudio: Bleda Gaming Studios

### ISSUE THREAD

**Issue by afk-mario** [indi-es/estudios#129]
## Gracias a: afk - afk@ellugar.co

```json
{
  "name": "Bleda Gaming Studios",
  "country": "México",
  "city": "CDMX",
  "state": "CDMX",
  "website": "https://netsites.mx/bledagamestudios/#xl_xr_page_index.html",
  "twitter": "",
  "facebook": "https://www.facebook.com/bledagamestudios/",
  "instagram": "",
  "tags": [],
  "inactive": false,
  "last_time_active": "2021-11-02T21:34:37.233Z"
}
```

**Comment by Ucamo** [comment]
Part of #133

**Comment by afk-mario** [comment]
closed #133
